rpc/query_server.go

```diff
@@ -268,7 +269,9 @@ func (s *QueryServer) queryEvm(state loomchain.State, caller, contract loom.Addr
 			return nil, err
 		}
 	}
-	vm := levm.NewLoomVm(state, nil, nil, createABM, false)
+	version := state.Block().Height
+	evmSnapshot := s.EvmStore.GetSnapshot(version).(*store.EvmStoreSnapshot)
```

Comment:
This casting looks silly, why doesn't `EvmStore.GetSnapshot()` return `*store.EvmStoreSnapshot` in the first place?

Reply:
Fixed